src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [this.a, this.b, this.c];
+    const sortedSide = sides.sort((x, y) => y - x);
```

Comment:
```suggestion
    const sortedSide = sides.sort((x, y) => y - x);
```
It is useless to create this variable if you don't make a copy of sides array because of sort mutate an array